sanskrit_parser/base/sanskrit_base.py

```diff
@@ -62,7 +62,8 @@ class SanskritObject(object):
 
     """
 
-    def __init__(self, thing=None, encoding=None, unicode_encoding='utf-8', strict_io=True):
+    def __init__(self, thing=None, encoding=None, unicode_encoding='utf-8',
+                 strict_io=True, strict_io_vis=True):
```

Comment:
I was a bit confused with the naming. Does strict_io_vis mean visargas at the end are "strict" i.e. not replaced, or they are to be replaced? Intuitively, I would have thought it's the former (i.e. they should not be replaced) but looks like they are the latter (i.e. will be replaced). Can we rename this to be "replace_vis_end" or "preserve_vis_end" so that it is clear what this param does.